bindings/gumjs/gumdukinterceptor.c

```diff
@@ -193,6 +203,8 @@ _gum_duk_interceptor_init (GumDukInterceptor * self,
   self->invocation_args = _gumjs_duk_require_heapptr (ctx, -1);
   duk_put_global_string (ctx, "InvocationArgs");
   // []
+  self->cached_invocation_args = gumjs_invocation_args_new (ctx, NULL, self);
+  self->cached_invocation_args_in_use = FALSE;
```

Comment:
Sorry, missed one detail. We should move the initialization of `cached_invocation_context*` and `cached_invocation_args*` to the end of this function, separated by blank lines, so this caching aspect is visually isolated (like in the struct).